en/release-notes.md

```diff
@@ -1,5 +1,28 @@
 ## Notification > Email > Release Notes
 
+### 2024. 08. 27.
+
+#### 기능 개선/변경
+
+* [Console] 통계 이벤트 처리 개선
+    * 이메일 발송 중 중복 처리하던 통계 데이터를 중복으로 처리하지 않도록 개선했습니다.
+    * 통계 이벤트 중 '수신' 이벤트를 '수신'과 '수신 실패'로 분류했습니다.
+        * '수신' 이벤트는 수신자에게 메일이 성공적으로 전달된 경우입니다.
+        * '수신 실패' 이벤트는 수신자에게 메일이 전달되지 않은 경우입니다. (예: Soft Bounce, Hard Bounce)
+
+* [Console] 템플릿 미리보기 기능 개선
+    * FreeMarker 템플릿의 미리보기 기능이 개선되었습니다.
+    * 템플릿 파라미터 적용 시 일부 파라미터가 적용되지 않을 경우 빈 값으로 표시되고 미리보기를 확인할 수 있습니다.
+
+* [Console] DNS 질의 실패시 발생하는 에러 코드 추가
+    * '도메인 소유 인증', 'SPF 인증', 'DKIM 인증', 'DMARC 인증' 과정에서 DNS 질의가 실패할 경우, 명확한 원인을 파악할 수 있도록 새로운 에러 코드가 추가되었습니다.
+    * 이는 문제 해결에 필요한 정보를 더 효과적으로 제공하여 개발자와 운영자가 DNS 관련 문제를 신속하게 진단하고 대응할 수 있도록 돕기 위함입니다.
+    * 주요 변경 사항
+        * 에러 코드: -2726
+        * 에러 메시지: DNS LookUp failed. lookup failed message: {}
+            * {} 안에는 DNS 질의시 발생한 오류 메시지가 포함되어, 구체적인 실패 원인을 확인할 수 있습니다. 예를 들어, 도메인 오기입, 네트워크 오류, 텍스트 레코드 오기입 등이 원인일 수 있습니다.
+    * 발생한 에러 코드는 개발자 도구를 통해 실시간으로 확인할 수 있습니다.
+
```

Comment:
```suggestion
### August 27, 2024

#### Feature Updates

* [Console] Improved statistical event handling
    * Improved the duplicate processing of statistical data during email sending.
    * Categorized the 'Receive' events into 'Received' and 'Failed to receive'.
        * ‘Received' event is when email has been successfully delivered to the recipient.
        * ‘Failed to receive’ event is when email is not delivered to the recipient (e.g., Soft Bounce, Hard Bounce).

* [Console] Improved template preview feature.
    * Improved preview feature for FreeMarker templates.
    * When applying template parameters, if some parameters are not applied, they appear as empty values and preview is available.

* [Console] Added error codes for failed DNS queries
    * Added new error codes for DNS queries that fail during Domain Ownership Authentication, SPF Authentication, DKIM Authentication, and DMARC Authentication to provide a clearer indication of the cause.
    * This is to help developers and operators quickly diagnose and respond to DNS-related issues by better providing the information needed for troubleshooting.
    * Major changes
        * Error code: -2726
        * Error message: DNS LookUp failed. lookup failed message: {}
            * {} contains the error message that occurred during the DNS query, so you can determine the specific reason for the failure. For example, it could be a domain misspelling, a network error, or a text record misspelling.
    * You can see the error code in real time through the developer tools.
    
```
@gehwan96 영어 번역 전달드립니다.